### PR TITLE
Docs: minimal docs for links preview addons

### PR DESCRIPTION
**Pull request by humitos**
Reference https://github.com/readthedocs/ext-theme/pull/521

---
:books: Documentation previews :books:

- User's documentation (`docs`): https://docs--11828.org.readthedocs.build/en/11828/

- Developer's documentation (`dev`): https://dev--11828.org.readthedocs.build/en/11828/

### CODE REVIEW

File: docs/user/addons.rst [hunk 1]
```diff
@@ -5,9 +5,12 @@ Read the Docs Addons
 They are used in the rendered documentation,
 and can be accessed via hotkeys or on screen UI elements.
 
-:doc:`DocDiff </pull-requests>`
```
Comment (ericholscher) [inline docs/user/addons.rst:8]: 🙃

File: docs/user/links-preview.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Links preview
+=============
```
Comment (ericholscher) [inline docs/user/links-preview.rst:2]:
```suggestion
Link previews
=============
```

File: docs/user/links-preview.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Links previews allows you to see the content of the link you are about to click when you hover it.
+This content will be rendered inside a popup that disappears after moving the mouse outside of it.
```
Comment (ericholscher) [inline docs/user/links-preview.rst:5]:
```suggestion
Link previews allows you to see the content of the link you are about to click when you hover over it.
This gives you the ability to quickly preview where a link points,
so you can keep your context while reading but learn more from other resources. 
```

I think it's useful to give the "why should I care?" to the user in the first couple sentences.

File: docs/user/links-preview.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+.. figure:: /_static/images/addons-links-preview.png
+   :width: 80%
+
+   Example of links preview addon
```
Comment (ericholscher) [inline docs/user/links-preview.rst:10]:
```suggestion
   Example of link previews addon
```

File: docs/user/links-preview.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+
+
+Enabling links previews
+-----------------------
```
Comment (ericholscher) [inline docs/user/links-preview.rst:15]:
```suggestion
Enabling link previews
----------------------
```

File: docs/user/links-preview.rst [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+In your dashboard, you can go to the links preview tag in :guilabel:`Settings > Addons > Links preview` and enable it.
```
Comment (ericholscher) [inline docs/user/links-preview.rst:17]:
```suggestion
In your dashboard, you can go to the links preview tag in :guilabel:`Settings > Addons > Link previews` and enable it.
```

File: docs/user/links-preview.rst [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+Troubleshooting links previews
+------------------------------
```
Comment (ericholscher) [inline docs/user/links-preview.rst:20]:
```suggestion
Troubleshooting link previews
-----------------------------
```

**Review by ericholscher — APPROVED** [review]

File: docs/user/links-preview.rst (hunk 2 above)
Comment (humitos) [reply docs/user/links-preview.rst:2]: Ouch, I've been using "Links preview" everywhere 🙃

**Pull request merged**